controllers/vrg_volrep.go

```diff
@@ -1913,19 +1922,13 @@ func (v *VRGInstance) restorePVsAndPVCsFromS3(result *ctrl.Result) (int, error)
 func (v *VRGInstance) restorePVsFromObjectStore(objectStore ObjectStorer, s3ProfileName string) (int, error) {
 	pvList, err := downloadPVs(objectStore, v.s3KeyPrefix())
 	if err != nil {
-		v.log.Error(err, fmt.Sprintf("error fetching PV cluster data from S3 profile %s", s3ProfileName))
-
 		return 0, err
 	}
 
 	v.log.Info(fmt.Sprintf("Found %d PVs in s3 store using profile %s", len(pvList), s3ProfileName))
 
 	if err = v.checkPVClusterData(pvList); err != nil {
-		errMsg := fmt.Sprintf("Error found in PV cluster data in S3 store %s", s3ProfileName)
-		v.log.Info(errMsg)
-		v.log.Error(err, fmt.Sprintf("Resolve PV conflict in the S3 store %s to deploy the application", s3ProfileName))
-
-		return 0, fmt.Errorf("%s: %w", errMsg, err)
+		return 0, err
```

Comment:
Add context?

    return 0, fmt.Errorf("error restoring PVs from profile %q: %w", s3ProfileName, err)

Reply:
Added context in caller function